server/src/test/java/org/elasticsearch/action/admin/indices/rollover/TransportRolloverActionTests.java

```diff
@@ -110,7 +110,8 @@ public class TransportRolloverActionTests extends ESTestCase {
         mockCreateIndexService,
         mdIndexAliasesService,
         EmptySystemIndices.INSTANCE,
-        WriteLoadForecaster.DEFAULT
+        WriteLoadForecaster.DEFAULT,
+        mockClusterService
```

Comment:
Could we also add a test case here to make sure we're covering the few changes in the transport action?

Reply:
What kind of test are you looking for exactly? My changes in `TransportRolloverAction` are basically just passing the `FailureStoreOptions` to the downstream methods - not any actual logic. So my thinking is that compilation acts as a partial test for this combined with the YAML REST test that makes sure everything ties in together neatly.

Reply:
I was thinking of a real small test just to make sure that we are indeed passing the value along, that way if we ever make a refactoring here we know we haven't broken the handoff but, yeah, considering its a small test and likely covered by the REST tests already, I'm fine if we skipped it in retrospect.